src/main/scala/org/broadinstitute/dsde/firecloud/core/ProfileClient.scala

```diff
@@ -340,14 +340,15 @@ class ProfileClientActor(requestContext: RequestContext) extends Actor with Fire
       case None => Put
     }
 
-    memberList flatMap { members => pipeline { rawlsRequestMethod(url, members) } }
-
-    // don't leak any sensitive data
-    RequestComplete(NoContent)
-  }.recover {
+    memberList flatMap { members => pipeline { rawlsRequestMethod(url, members) } } map { response =>
```

Comment:
this was the ideal approach but I don't think it's going to work well. this call will probably time out 100% of the time it's called. it needs to wait for rawls to issue all of the google group updates. for n changes to the whitelist, there will be n calls to google just to update the dbGapAuthorizedUsers group. Then, for each of the n users, we need to issue a call to google to update the intersection groups for each protected workspace that they're a member of. rawls also needs to update all of the groups in the database. this is probably the slowest endpoint we have.

an alternative approach is to undo these changes and go back to issuing the rawls call on the side and then return some status code like Accepted. in the background, we'll log whether or not the call to rawls succeeded. regardless of the approach, we also need to be aware of the fact that the call to rawls itself will likely timeout too.

thoughts?